### ISSUE TITLE
Completely hide scrollbar

### ISSUE THREAD

**Issue by coldpumpkin** [malihu/malihu-custom-scrollbar-plugin#261]
Is it possible to completely hide the scrollbar? Or, only show when the user passes the cursor on top or uses the mousewheel.

**Comment by malihu** [comment]
Set `autoHideScrollbar` option parameter to `true` 
[more info](http://manos.malihu.gr/jquery-custom-content-scroller/#plugin-data-autoHideScrollbar)

**Comment by coldpumpkin** [comment]
Yes, but I want it to be hidden also when the user has the scroll on top of the div...
